deploy/monitor/splunk/user-seed.conf

```diff
@@ -0,0 +1,3 @@
+[user_info]
+USERNAME = admin
+PASSWORD = password
```

Comment:
this username and password works?? haha

Reply:
This seeds the username and password into the UF. We don't care about them being secure because the UF is only being run on the instance we create.